README.md

```diff
@@ -17,12 +17,39 @@ Add this config in the dApp's `embark.json`:
   }
 ```
 ### Development environment
-To configure for a development environment, the blockchain client and the webserver need to be set up to accept outside connections.
+To configure for a development environment, the blockchain client, webserver, and storage need to be set up to accept outside connections.
 #### Blockchain client
 Set `config/blockchain.js > development > rpcHost` to `0.0.0.0`. This will open up your blockchain client (Geth or Parity) to outside connections.
 #### Webserver
 Set `config/webserver.js > host` to `0.0.0.0`. This will open up your webserver to outside connections.
 > NOTE: When the Status browser opens the dApp, it will open the IP of the machine running Embark along with the port specified in the webserver config, ie `http://192.168.0.15:8000`. This is so that the device can connect to the webserver started by Embark.
+#### Storage
+Our machine will be running our storage node, and we need to access the node from our dApp, so we have to configure IPFS to run on our machine's IP, then tell embark to access the node via the machine IP.
+> NOTE: `embark-status` has only been tested with IPFS. This guide assumes you have [ipfs installed](https://docs.ipfs.io/introduction/install/) and you are using it for your dApp.
+
+Set `config/storage.js > dappConnection` to
+```
+dappConnection: [
+  {
+    provider: "ipfs",
+    host: "your-machine-ip",
+    port: 5001,
+    getUrl: "http://your-machine-ip:8080/ipfs/"
+  }
+]
+```
+Then run the following commands:
+```
+ipfs config --json Addresses.API "\"/ip4/0.0.0.0/tcp/5001\""
+ipfs config --json Addresses.Gateway "\"/ip4/0.0.0.0/tcp/5001\""
+```
+> NOTE: These commands change your local IPFS config so that IPFS can be used on any IP. In this case, we want the IPFS API and gateway to the be accessible from our machine's IP.
```

Comment:
Isn't that a bit of a security issue? Should we open it only to our phone?

Reply:
Good catch! Updated.